neqo-transport/src/path.rs

```diff
@@ -979,6 +1007,25 @@ impl Path {
             self.rtt.pto(space), // Important: the base PTO, not adjusted.
             lost_packets,
         );
+
+        if self.is_ecn_enabled() {
+            // If the path is currently marking outgoing packets as ECT(0),
+            // update the count of lost ECN-marked bytes.
+            self.lost_ecn_bytes += lost_packets.iter().map(|p| p.size).sum::<usize>();
+
+            // If we lost more than 3 MTUs worth of ECN-marked bytes, then
+            // disable ECN on this path. See RFC 9000, Section 13.4.2.
+            // This doesn't quite implement the algorithm given in RFC 9000,
+            // Appendix A.4, but it should be OK. (It might be worthwhile caching
+            // destination IP addresses for paths on which we had to disable ECN,
+            // in order to not persitently delay connection establishment to
+            // those destinations.)
+            if self.lost_ecn_bytes > MAX_DATAGRAM_SIZE * 3 {
```

Comment:
It'd be good to have a test for this behavior.

Reply:
Good point, will add.